src/pint/pintk/plk.py

```diff
@@ -1249,7 +1249,7 @@ def coordToPoint(self, cx, cy):
             if dist[ind] > clickDist:
                 log.warning("Not close enough to a point")
                 ind = None
-        return index
+        return ind
```

Comment:
All I know is that @scottransom said this change would fix the problem and in my quick tests it did.

Reply:
It just looks risky - functions that call this could be expecting either the thing that was in the original TOA index column or the index into the current array of TOAs, and this change switches which this function returns.

Reply:
OK, maybe we should handle this in a separate PR.  I'll remove that change.